lib/rollbar/exception_reporter.rb

```diff
@@ -7,16 +7,17 @@ def report_exception_to_rollbar(env, exception)
 
       exception_data = exception_data(exception)
 
-      if exception_data.is_a?(Hash)
+      case exception_data
+      when Hash
         env['rollbar.exception_uuid'] = exception_data[:uuid]
         Rollbar.log_debug(
           "[Rollbar] Exception uuid saved in env: #{exception_data[:uuid]}"
         )
-      elsif exception_data == 'disabled'
+      when 'disabled'
```

Comment:
This change is not 100% equivalent as it expresses `exception_data === 'disabled'`. It's probably still correct, but just want to point it out. In case it requires updating, there are few more similar changes in this PR.

Reply:
Thanks for the comment. Strings are one of the few cases (pardon the pun) where `===` behaves the same as `==`. 

In fact, String `===` is an alias of `==`,  so it is literally the same.
https://github.com/ruby/ruby/blob/v2_7_3/string.c#L3253-L3277

Not so for many other types though.

Reply:
Thanks for the clarification!